docker-compose.yml

```diff
@@ -1,6 +1,21 @@
 version: '3'
 services:
+  test:
+    build: .
+    command: 'npm run test:docker:ci'
+    depends_on:
+      - rediscluster
+      - rabbitmq
+    volumes:
+      - .:/usr/windbreaker
+      - /usr/windbreaker/node_modules
```

Comment:
For a faster feedback loop, I recommend running `docker-compose run test bash` to get access to container itself. You can make changes from the host and run tests from within the container.